src/components/login/Login.jsx

```diff
@@ -0,0 +1,103 @@
+import MailOutlinedIcon from "@mui/icons-material/MailOutlined";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import * as yup from "yup";
+
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import "./Login.css";
+import {
+  Box,
+  Button,
+  Grid,
+  InputAdornment,
+  TextField,
+  Typography,
+} from "@mui/material";
+import google from "/src/imgs/gmail.png";
+const Login = () => {
+  const schema = yup.object().shape({
+    email: yup.string().email("it must be a e-mail").required("insert value"),
+    password: yup
+      .string()
+      .min(4, "It must have 4 characters")
+      .max(20, "It must be less than 20 characters")
+      .required("insert value"),
+  });
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(schema),
+  });
+
+  const onSubmit = (data) => {
+    console.log(data);
```

Comment:
you should never push a console log to the Pull Request. Please for now you should redirect to /home path withot checking user and password.